src/base/static/components/organisms/main-map.js

```diff
@@ -151,9 +151,11 @@ class MainMap extends Component {
     });
 
     // Handler for clearing in-progress drawing geometry.
-    this.props.router.on("route", () => {
-      this._map.drawDeleteGeometry();
-    });
+    if (this.props.mapConfig.options.drawing_enabled !== false) {
```

Comment:
We may want to consider centralizing the default values for our config, if we haven't already done so. Perhaps this is something that we can address when we convert the config to JSON.